components/FilterForm/FilterForm.js

```diff
@@ -41,7 +41,7 @@ function between(x, min, max) {
 }
 
 function FilterForm() {
-  const [list, setList] = useState(data);
+  const [list, setList] = useLocalStorageState(data, { defaultValue: [] });
```

Comment:
I think the first argument to useLocalStorageState needs to be the `key` and not the data. The second argument needs to be where we would put the data. i.e.

``` js
  const [list, setList] = useLocalStorageState("list", { defaultValue: data });
```